src/db/h2/lrsql/h2/result.clj

```diff
@@ -0,0 +1,41 @@
+(ns lrsql.h2.result
+  "Process raw h2 results for parity with other backends"
+  (:require [com.yetanalytics.squuid :as squuid]
+            [lrsql.util :as u]))
+
+(defn query-last-statement-stored-result
+  "Extracts a timestamp suitable for last-statement-stored from an H2 query
+  result. If result is nil, returns nil."
+  [query-result]
+  (when-let [{:keys [id]} query-result]
+    {:lstored (-> id squuid/uuid->time u/time->str)}))
+
+(defn query-platform-frequency-result
+  "Given a result of all statement payloads, return the frequencies of each"
+  [query-result]
+  (map
+   (fn [[platform scount]]
+     {:platform platform
+      :scount   scount})
+   (frequencies
```

Comment:
This is a core function? 🤯

Reply:
Haha yeah the usual philosophy of keeping the core small can take a backseat to things that are just hugely useful to the working programmer and I'm here for it.

Reply:
If only I knew, would've been useful for, say, ID counting in Pan.